### PR TITLE
K8s deploy config templates example

### PR DESCRIPTION
**Pull request by eysi09**
**What this PR does / why we need it**:

The previous config template example was quite basic and missing a proper README.

This example builds on the K8s shared manifests for consistency.

In follow up commits I'll remove the old example and update our docs to
point to the new example.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by worldofgeese — CHANGES_REQUESTED** [review]
I've confirmed this example runs locally. Asked for a few changes.

File: examples/k8s-deploy-config-templates/README.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+  - kind: Deploy
+    type: kubernetes
+    name: ${parent.name}
+
+    spec:
+      files:
+        - manifests/deployment.yaml
+        - manifests/service.yaml
+        - "${inputs.enableIngress ? 'manifests/ingress.yaml' : null }" # <--- The "inputs" are defined in the JSON schema referenced in the 'inputsSchemaPath' field
```
[the remark this answers is not in the document]
Comment (eysi09) [reply examples/k8s-deploy-config-templates/README.md:39]: I don't that think that works inside inline code like this. Or am I missing something?

I can just remove this part of the sample config though if you think that's better.

File: examples/k8s-deploy-config-templates/web/README.md
[the remark this answers is not in the document]
Comment (eysi09) [reply examples/k8s-deploy-config-templates/web/README.md:1]: Nope, I'll remove it.

**Review by worldofgeese — APPROVED** [review]
This looks ready to fly 🕊️ to me! Thanks for addressing.

**Pull request merged**